### ISSUE TITLE
Browser grey lines make text hard to parse in night mode and text aligment

### ISSUE THREAD

**Issue by voidplayer** [ankidroid/Anki-Android#5707]
This is an example of the current state:

Its not so bad when its light...
![bad-white](https://user-images.githubusercontent.com/4019910/72111425-0a78d700-333b-11ea-9856-4a6cccfbecd5.png)

... But Its specially bad when in black mode:

![bad-black](https://user-images.githubusercontent.com/4019910/72111424-0a78d700-333b-11ea-897f-f6013c2572c9.png)

Theres a couple of usability issues here:
* Browser shows a couple of fields. The size is determined by the largest field and the other field is centered vertically
* line is small and light grey. Grey is a bad color for contrast. Contrast is something desirable when visually parsing data

This is a mockup of how it could be better, although im no designer:

Thicker white line top aligned
![good-black1](https://user-images.githubusercontent.com/4019910/72111423-0a78d700-333b-11ea-9a71-e99afcf8f4ec.png)

Alternating colors top aligned
![good-black2](https://user-images.githubusercontent.com/4019910/72111422-09e04080-333b-11ea-805c-50908ffa344b.png)

Thanks for considering!!

**Comment by david-allison-1** [comment]
Moving off stale - it's useful and would be good as a first issue. I'd like to see this

**Comment by voidplayer** [comment]
👋

**Comment by david-allison-1** [comment]
Thanks for your patience @voidplayer 

I guess "Be the change you want to see in the world" is the quote of the day (by Arleen Lorrance) 

I'll get this looked at today

Can't go with alternating colours as that would obscure the other colours that we already use - will try for the solid white line.

**Comment by voidplayer** [comment]
No, thank you for looking at it

Anything should be better than the current thin grey line. Grey is for unimportant things that can be skipped (side notes, copyright,...) not for an item visual limits

I think the horizontal line should be a little thicker than the font. Top alignment of the items to easily find where they start. Maybe even a thin vertical between the 2 fields to help parse the

Thin because they belong together, in contract with thicc where they are separated items

Again, im no designer, i just have eyes

(Meaning im not sure what is the optimal approach to make it look fancy while allowing it to do its job 😬)

**Comment by david-allison-1** [comment]
@voidplayer 

First round (before I read the comments), I personally prefer a thinner line, as even a 1dp line seems jarring

![image](https://user-images.githubusercontent.com/62114487/89406868-b6d63c80-d715-11ea-9aaf-1593a96ee162.png)

**Comment by voidplayer** [comment]
The problem with your test case is that it is very easy to parse already. Almost all fields are 1 line, with little text

I would try it with a real world deck :)

**Comment by david-allison-1** [comment]
Better?

![image](https://user-images.githubusercontent.com/62114487/89408400-3bc25580-d718-11ea-9f9e-eb5287005970.png)

**Comment by voidplayer** [comment]
i am the very model of a modern major general?😆 (yes, i searched the video and saw the song)

And yes, anything is better than the current one, so very pleased if you just push whatever you can come up with even if it can still be improved in the future :)

Something is still off. I think is the gap between words being similar sized than the vertical gap between question and answer

When theres lot of text on the same side, the body of the text visually create something similar to a line. But when you mix and match fields that are differently sized, this wall is no more. (look at my first mock up for reference of alternating bigger fields left and right)

I would try an actual line, slightly thinner

Also, i would try to grey very, very slightly the text to see if there is an improvement on reading. My reasoning is that the line helps you put your eyes on the word. Once is there, you can easily keep reading if its what you are looking for or discard and go to the next line if its not, even if the text is slightly greyed

Not sure if it will actually work tho :)

**Comment by david-allison-1** [comment]
Take 3: added vertical bar, modified margins:

![image](https://user-images.githubusercontent.com/62114487/89413633-0f5f0700-d721-11ea-9105-389b7b506cbf.png)

**Comment by voidplayer** [comment]
I like it thanks! :D
